reacton/generate_test.py

```diff
@@ -22,7 +25,6 @@ def _MyTest(
 
 @implements(_MyTest)
 def MyTest(**kwargs):
-
```

Comment:
@maartenbreddels Very close, but not quite, seems that when using ruff this initial space is removed, but that seems to be the only difference, at least in our test cases.